### ISSUE TITLE
is invite fails on current dev-build

### ISSUE THREAD

**Issue by rlf** [rlf/uSkyBlock#372]
I find another problem. If we invite a player who already owns an island, he loses his island but he's not invited on the new island.
```
[01:21:26] [Server thread/INFO]: mxlive24 issued server command: /is invite CitoyenCharly
[01:21:29] [Server thread/INFO]: CitoyenCharly issued server command: /is accept
[01:21:40] [Server thread/INFO]: CitoyenCharly issued server command: /is h
[01:21:47] [Server thread/INFO]: [uSkyBlock] Cleared island in 17.611 seconds
[01:21:47] [Server thread/WARN]: [uSkyBlock] Task #91840 for uSkyBlock vCielbloc generated an exception
java.lang.NullPointerException
at us.talabrek.ultimateskyblock.player.PlayerLogic.getPlayerInfo(PlayerLogic.java:39) ~[?:?]
at us.talabrek.ultimateskyblock.uSkyBlock.getPlayerInfo(uSkyBlock.java:893) ~[?:?]
at us.talabrek.ultimateskyblock.command.InviteHandler.addPlayerToParty(InviteHandler.java:134) ~[?:?]
at us.talabrek.ultimateskyblock.command.InviteHandler$2.run(InviteHandler.java:113) ~[?:?]
at us.talabrek.ultimateskyblock.uSkyBlock$4.run(uSkyBlock.java:510) ~[?:?]
at us.talabrek.ultimateskyblock.handler.WorldEditHandler$1.run(WorldEditHandler.java:159) ~[?:?]
at org.bukkit.craftbukkit.v1_8_R2.scheduler.CraftTask.run(CraftTask.java:71) ~[spigot.jar:git-Spigot-2ec6f06-7722428]
at org.bukkit.craftbukkit.v1_8_R2.scheduler.CraftScheduler.mainThreadHeartbeat(CraftScheduler.java:350) [spigot.jar:git-Spigot-2ec6f06-7722428]
at net.minecraft.server.v1_8_R2.MinecraftServer.A(MinecraftServer.java:720) [spigot.jar:git-Spigot-2ec6f06-7722428]
at net.minecraft.server.v1_8_R2.DedicatedServer.A(DedicatedServer.java:368) [spigot.jar:git-Spigot-2ec6f06-7722428]
at net.minecraft.server.v1_8_R2.MinecraftServer.z(MinecraftServer.java:651) [spigot.jar:git-Spigot-2ec6f06-7722428]
at net.minecraft.server.v1_8_R2.MinecraftServer.run(MinecraftServer.java:554) [spigot.jar:git-Spigot-2ec6f06-7722428]
at java.lang.Thread.run(Thread.java:745) [?:1.7.0_79]
[01:21:53] [Server thread/INFO]: CitoyenCharly issued server command: /is h
[01:21:53] [Server thread/ERROR]: null
org.bukkit.command.CommandException: Unhandled exception executing command 'is' in plugin uSkyBlock vCielbloc
at org.bukkit.command.PluginCommand.execute(PluginCommand.java:46) ~[spigot.jar:git-Spigot-2ec6f06-7722428]
at org.bukkit.command.SimpleCommandMap.dispatch(SimpleCommandMap.java:141) ~[spigot.jar:git-Spigot-2ec6f06-7722428]
at org.bukkit.craftbukkit.v1_8_R2.CraftServer.dispatchCommand(CraftServer.java:646) ~[spigot.jar:git-Spigot-2ec6f06-7722428]
at net.minecraft.server.v1_8_R2.PlayerConnection.handleCommand(PlayerConnection.java:1139) [spigot.jar:git-Spigot-2ec6f06-7722428]
at net.minecraft.server.v1_8_R2.PlayerConnection.a(PlayerConnection.java:974) [spigot.jar:git-Spigot-2ec6f06-7722428]
at net.minecraft.server.v1_8_R2.PacketPlayInChat.a(PacketPlayInChat.java:45) [spigot.jar:git-Spigot-2ec6f06-7722428]
at net.minecraft.server.v1_8_R2.PacketPlayInChat.a(PacketPlayInChat.java:1) [spigot.jar:git-Spigot-2ec6f06-7722428]
at net.minecraft.server.v1_8_R2.PlayerConnectionUtils$1.run(SourceFile:13) [spigot.jar:git-Spigot-2ec6f06-7722428]
at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:471) [?:1.7.0_79]
at java.util.concurrent.FutureTask.run(FutureTask.java:262) [?:1.7.0_79]
at net.minecraft.server.v1_8_R2.SystemUtils.a(SourceFile:60) [spigot.jar:git-Spigot-2ec6f06-7722428]
at net.minecraft.server.v1_8_R2.MinecraftServer.A(MinecraftServer.java:712) [spigot.jar:git-Spigot-2ec6f06-7722428]
at net.minecraft.server.v1_8_R2.DedicatedServer.A(DedicatedServer.java:368) [spigot.jar:git-Spigot-2ec6f06-7722428]
at net.minecraft.server.v1_8_R2.MinecraftServer.z(MinecraftServer.java:651) [spigot.jar:git-Spigot-2ec6f06-7722428]
at net.minecraft.server.v1_8_R2.MinecraftServer.run(MinecraftServer.java:554) [spigot.jar:git-Spigot-2ec6f06-7722428]
at java.lang.Thread.run(Thread.java:745) [?:1.7.0_79]
Caused by: java.lang.NullPointerException
at us.talabrek.ultimateskyblock.player.PlayerLogic.getPlayerInfo(PlayerLogic.java:39) ~[?:?]
at us.talabrek.ultimateskyblock.uSkyBlock.getPlayerInfo(uSkyBlock.java:893) ~[?:?]
at us.talabrek.ultimateskyblock.command.island.RequireIslandCommand.doExecute(RequireIslandCommand.java:38) ~[?:?]
at us.talabrek.ultimateskyblock.command.island.RequirePlayerCommand.execute(RequirePlayerCommand.java:31) ~[?:?]
at us.talabrek.ultimateskyblock.command.common.CompositeUSBCommand.execute(CompositeUSBCommand.java:117) ~[?:?]
at us.talabrek.ultimateskyblock.command.common.AbstractCommandExecutor.onCommand(AbstractCommandExecutor.java:21) ~[?:?]
at us.talabrek.ultimateskyblock.command.IslandCommand.onCommand(IslandCommand.java:71) ~[?:?]
at org.bukkit.command.PluginCommand.execute(PluginCommand.java:44) ~[spigot.jar:git-Spigot-2ec6f06-7722428]
... 15 more
```

**Comment by dutchy1001** [comment]
Invite still fails, the invited person does loose their island and is teleported to the spawn and is not in the party of who invited the player. 
The invited players island is not deleted but the WG region file is has the entrée removed.
The orphan file is updated with the invited players location to be reused.
The players file states "hasIsland: false" which is correct.
The island folder has no entrée for the island which is also correct.

**Comment by rlf** [comment]
Related to the async loading of player-data, i.e. same root-cause as `/is home` that will fail if the player hasn't accessed `/is` before executing it.

**Comment by dutchy1001** [comment]
this is fixed in v2.3-hf8d
